packages/router/src/lib/operations/cancel.ts

```diff
@@ -12,6 +12,9 @@ import { getContext } from "../../router";
 import { ParamsInvalid, ReceiverTxExists } from "../errors";
 import { CancelInput, CancelInputSchema } from "../entities";
 import { TransactionStatus } from "../../adapters/subgraph/graphqlsdk";
+import { SenderTxTooNew } from "../errors/cancel";
+
+export const SENDER_PREPARE_BUFFER_TIME = 60 * 5; // 5 mins
```

Comment:
bsc block time is 3s, we usually have an out of sync buffer of up to 250. this is quite long enough imo